Test failures on Python 3.3 only, but they look real:

```
ERROR: IPython.html.tests.test_serialize.test_serialize_binary
----------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.3.5/lib/python3.3/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.3.5/lib/python3.3/site-packages/IPython/html/tests/test_serialize.py", line 17, in test_serialize_binary
    bmsg = serialize_binary_message(msg)
  File "/home/travis/virtualenv/python3.3.5/lib/python3.3/site-packages/IPython/html/base/zmqhandlers.py", line 56, in serialize_binary_message
    return b''.join(buffers)
TypeError: sequence item 2: expected bytes, memoryview found
```

--- a/IPython/html/base/zmqhandlers.py
+++ b/IPython/html/base/zmqhandlers.py
@@ -8,6 +8,7 @@ import os
 import json
 import struct
 import warnings
+import sys
 
 try:
     from urllib.parse import urlparse # Py 3
@@ -43,8 +44,8 @@ def serialize_binary_message(msg):
     # don't modify msg or buffer list in-place
     msg = msg.copy()
     buffers = list(msg.pop('buffers'))
-    # for python 2, copy the buffer memoryviews to byte strings
-    buffers = [x.tobytes() for x in buffers]
+    if sys.version_info < (3, 4):
+        buffers = [x.tobytes() for x in buffers]
     bmsg = json.dumps(msg, default=date_default).encode('utf8')
     buffers.insert(0, bmsg)
     nbufs = len(buffers)

